redstone/client.py

```diff
@@ -598,13 +620,247 @@ def unwrap(self, key_id, ciphertext, aad=None):
         resp = self._action(key_id, "unwrap", data)
         return base64.b64decode(resp["plaintext"].encode("utf-8"))
 
+    def rewrap(self, key_id, ciphertext, aad=None):
+        # json body needs to be a UTF-8 string
+        if isinstance(ciphertext, bytes):
+            ciphertext = ciphertext.decode("utf-8")
+
+        data = {"ciphertext": ciphertext}
+
+        if aad:
+            data["aad"] = aad
+
+        return self._action(key_id, "rewrap", data)
+
     def rotate(self, key_id, payload=None):
         data = None
         if payload:
             data = {"payload": base64.b64encode(payload).decode("utf-8")}
 
         return self._action(key_id, "rotate", data)
 
+    def set_key_for_delete(self, key_id):
+        # authorize deletion for a key with a dual authorization policy
+        resp = self.session.post("%s/api/v2/keys/%s/actions/setKeyForDeletion" % (self.endpoint_url, key_id))
+        self._validate_resp(resp)
+
+    def unset_key_for_delete(self, key_id):
+        # remove an authorization for a key with a dual authorization policy
+        resp = self.session.post("%s/api/v2/keys/%s/actions/unsetKeyForDeletion" % (self.endpoint_url, key_id))
+        self._validate_resp(resp)
+
+    def disable(self, key_id):
+        resp = self.session.post("%s/api/v2/keys/%s/actions/disable" % (self.endpoint_url, key_id))
+        self._validate_resp(resp)
+
+    def enable(self, key_id):
+        resp = self.session.post("%s/api/v2/keys/%s/actions/enable" % (self.endpoint_url, key_id))
+        self._validate_resp(resp)
+
+    def get_registrations(self, key_id, crn=None):
+        # retrieves a list of registrations that are associated with a specified root key.
```

Comment:
```suggestion
        """
        Retrieve a list of registrations
        
        If `key_id` is None (the default) all registrations for the instance are returned, otherwise
        only the registrations for the associated with a specified root key are returned.
        """
```